css/deep-linking/_mobile.scss

```diff
@@ -39,21 +61,27 @@
         }
     }
 
-    &__logo {
-        height: 108px;
-        width: 77px;
+    &__href {
+        height: 2.2857142857142856em;
+        line-height: 2.2857142857142856em;
+        margin: 18px auto 20px;
+        max-width: 300px;
+        width: auto;
+        font-weight: bolder;
     }
 
     &__button {
         border: 0;
         height: 2.2857142857142856em;
         line-height: 2.2857142857142856em;
```

Comment:
Man, where did these exact numbers even come from? Do you know?

Reply:
No idea! Should I round them?

Reply:
No, because you didn't introduce them and it's not directly related to your changes. I was hoping you'd know.